src/tests/ftest/soak/soak_test_base.py

```diff
@@ -413,19 +521,28 @@ def run_soak(self, test_param):
             # Start new pass
             start_loop_time = time.time()
             self.log.info(
-                "<<Soak1 PASS %s: time until done %s>>", self.loop,
+                "<<Soak PASS %s: time until done %s>>", self.loop,
                 DDHHMMSS_format(self.end_time - time.time()))
-            # Create all specified pools
-            add_pools(self, pool_list)
+            # Create pool for jobs
+            add_pools(self, ["pool_jobs"])
             self.log.info(
                 "Current pools: %s",
                 " ".join([pool.uuid for pool in self.pool]))
+            # Initialize if harassers
```

Comment:
This comment doesn't really make sense. Typo?

Reply:
It should say Initialize if harassers enabled